src/linters/stylelint/index.js

```diff
@@ -0,0 +1,112 @@
+const Module = require( 'module' );
+const path = require( 'path' );
+const moduleAlias = require( 'module-alias' );
+
+/**
+ * Convert an error message from the stylelint format into one acceptable for GitHub
+ *
+ * @param {Object} message Data about a warning from stylelint.
+ * @returns {Object}
+ */
+const formatMessage = message => ( {
+	line:     message.line,
+	column:   message.column,
+	severity: message.severity,
+	message:  message.text,
+	source:   message.rule,
+} );
+
+/**
+ * Fetch a count of the total errors and warnings for our response.
+ *
+ * @param {Object} files Formatted warnings against specific files.
+ * @returns {{warnings: number, errors: number}}
+ */
+const getTotals = ( files ) => {
+	const allErrors = Object.keys( files )
+		.reduce( ( accumulator, key ) => accumulator.concat( [ ...files[key] ] ), [] );
```

Comment:
This isn't using .reduce properly, as it's mutating the accumulator rather than returning a new one.

Suggest replacing with:
```suggestion
	const allErrors = Object.keys( files ).reduce( ( acc, key ) => [ ...acc, ...files[key] ], [] );
```